client/functions.lua

```diff
@@ -918,3 +946,65 @@ function QBCore.Functions.StartParticleOnEntity(dict, ptName, looped, entity, bo
     end
     return particleHandle
 end
+
+function QBCore.Functions.GetStreetNametAtCoords(coords)
+    local streetname1, streetname2  	= GetStreetNameAtCoord(coords.x, coords.y, coords.z)
+    return { main = GetStreetNameFromHashKey(streetname1), cross = GetStreetNameFromHashKey(streetname2) }
+end
+
+function QBCore.Functions.GetZoneAtCoords(coords)
+    return GetLabelText(GetNameOfZone(coords))
+end
+
+function QBCore.Functions.GetCardinalDirection(entity)
+    entity = DoesEntityExist(entity) and entity or PlayerPedId()
+    if DoesEntityExist(entity) then
+        local heading = GetEntityHeading(entity)
+        if ((heading >= 0 and heading < 45) or (heading >= 315 and heading < 360)) then
+            return "North"
+        elseif (heading >= 45 and heading < 135) then
+            return "West"
+        elseif (heading >= 135 and heading < 225) then
+            return "South"
+        elseif (heading >= 225 and heading < 315) then
+            return "East"
+        end
+    else
+        return "Cardinal Direction Error"
+    end
+end
+
+function QBCore.Functions.GetCurrentTime()
+    local hour      = GetClockHours()
+    local minute    = GetClockMinutes()
+    local obj       = {}
+
+    if hour <= 12 then
+        obj.ampm = "AM"
+    elseif hour >= 13 then
+        obj.ampm = "PM"
+        hour = hour - 12
+    end
+
+    if minute <= 9 then
+        minute = "0" .. minute
+    end
+
+    obj.hour    = hour
+    obj.minute  = minute
+
+    return obj
+end
+
+function QBCore.Functions.GetGroundZCoord(coords)
+    if coords then
```

Comment:
Change this to `if not coords then return end` to reduce indentation

Reply:
Fixed :)